node_modules/oae-activity/tests/test-email.js

```diff
@@ -855,6 +857,59 @@ describe('Activity Email', function() {
     });
 
     /**
+     * Test that verifies that activity emails are not delivered to users who have deleted their
+     * profiles
+     */
+    it('verify email is not delivered to users who have since deleted their profile', function(callback) {
```

Comment:
Somehow I feel like we said we weren't going to do this for the group delete functionality because of performance drawbacks. Same with permissions.

@nicolaasmatthijs Should I just go ahead and just implement both of those?